feat: Workflow "latest crawl" tab

SuaYoo opened the pull request:
Resolves https://github.com/webrecorder/browsertrix/issues/2603

## Changes

- Combines "Watch" and "Logs" into single "Latest Crawl" tab
- Updates workflow routes and adds redirects
- Enables replaying and downloading latest crawl from the workflow detail view
- Tweaks crawl list table header labels and and archived item download button labels for consistency
- Fixes crawl queue showing error when stopping crawl

## Manual testing

To test the UI:
1. Log in as crawler
2. Click "Crawling"
3. Click complete workflow. Verify navigation to "Latest Crawl" tab and replay renders.
4. Click cloud icon download button. Verify crawl is downloaded as WACZ.
5. Click "Logs". Verify logs are disabled and downloadable if there are any.
6. Click "Run Crawl". Verify Replay tab changes to "Watch"
7. Wait for crawl to start. Verify watch renders as expected
8. Wait for crawl to finish. Verify replay renders as expected

To test redirects, verify:
- `/workflows/<id>#watch` redirects to `/latest`
- `/workflows/<id>#logs` redirects to `/logs`
- `/workflows/<id>#crawls` redirects to `/crawls`
- `/workflows/<id>#settings` redirects to `/settings`

## Screenshots

| Page | Image/video |
| ---- | ----------- |
| Workflow (side nav) | <img width="288" alt="Screenshot 2025-05-13 at 10 09 08 AM" src="https://github.com/user-attachments/assets/87b33558-ce20-4daf-b481-cb5c4a4b790b" /> |
| Workflow - Latest Crawl | **No crawls:**<br/><img width="973" alt="Screenshot 2025-05-13 at 10 10 14 AM" src="https://github.com/user-attachments/assets/6334b318-d5d0-4129-84c1-ca3bfa9c3bbb" /> |
| Workflow - Latest Crawl |  **Canceled crawl:**<br/><img width="985" alt="Screenshot 2025-05-13 at 10 30 58 AM" src="https://github.com/user-attachments/assets/130c7861-252b-474e-ad8a-296f1c713832" /> |
| Workflow - Latest Crawl | **Running crawl:**<br/><img width="972" alt="Screenshot 2025-05-13 at 10 50 55 AM" src="https://github.com/user-attachments/assets/fe886d40-78fa-4fd0-9a8c-29fd33b5a4cb" /> |
| Workflow - Latest Crawl | **Finished crawl:**<br/><img width="963" alt="Screenshot 2025-05-13 at 10 48 14 AM" src="https://github.com/user-attachments/assets/55fc2f97-a106-4548-af4b-facee3f3531a" /> |
| Workflow - Logs | <img width="960" alt="Screenshot 2025-05-13 at 10 10 04 AM" src="https://github.com/user-attachments/assets/532ff73f-bc37-439e-95de-198772c7168b" /> |



<!-- ## Follow-ups -->

SuaYoo committed:
update tab routes

SuaYoo committed:
update tab panels

SuaYoo committed:
clean up errors

SuaYoo committed:
render replay

SuaYoo committed:
enable download

SuaYoo committed:
update collection id

SuaYoo committed:
adjust states

SuaYoo committed:
fix crawl queue error

SuaYoo committed:
update redirect

tw4l commented:
Overall looks great! One quick usability thing I've noticed is that with this change we no longer have a quick link to QA the most recent crawl - maybe we could add that somewhere in the Latest Crawl tab when a crawl isn't active? An icon next to the download button in the Replay tab perhaps?

emma-sg commented:
Looks good overall! One thing re: download buttons/icons is that the tooltips don't actually say what the icons do — I might suggest using the tooltips for more direct labels (e.g. "Download Crawl"), and then details could be smaller text below? We've done a similar thing in the admin org list, off the top of my head: 
<img width="319" alt="Screenshot 2025-05-13 at 4 39 49 PM" src="https://github.com/user-attachments/assets/be503960-df5b-478f-9501-c7345c795b4d" />

emma-sg commented on frontend/src/pages/org/workflow-detail.ts:
```diff
@@ -438,153 +424,92 @@ export class WorkflowDetail extends BtrixElement {
+            WorkflowTab.LatestCrawl,
+            () =>
+              this.workflow?.isCrawlRunning
+                ? html`<btrix-badge variant="success">
+                    ${msg("Active")}
+                  </btrix-badge>`
```
Could we include the `btrix-crawl-status` here? I guess this might get confusing once there's pause states in play, but it'd be nice to maybe build the association between the icons we use and their labels

SuaYoo replied:
Do you mean duplicate the status shown above, or replace it?

<img width="354" alt="Screenshot 2025-05-13 at 2 07 46 PM" src="https://github.com/user-attachments/assets/54b9ff27-03f8-491d-b69e-4dfc93733d85" />

emma-sg replied:
Oh, I guess that's true — what about just the icon, then?

emma-sg replied:
I'm down to leave this as-is for now too though, especially considering paused states add more nuance than just "active" or "not active", so there might be several icons (in different colours) that would need to work against the cyan background too.

SuaYoo committed:
update tooltips

SuaYoo committed:
show qa rating

SuaYoo commented:
Updated download tooltips and added QA rating:

<img width="966" alt="Screenshot 2025-05-13 at 2 27 13 PM" src="https://github.com/user-attachments/assets/339eb8ff-e913-4323-8ded-fa23392f8625" />

SuaYoo committed:
get initial log totals

emma-sg reviewed (approved):
Looks good! As I mentioned, I think maybe leaving the "active" label for now is fine — maybe we can revisit when dealing with pause states later on.
